x-pack/plugins/observability_onboarding/public/components/app/header_action_menu/index.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiButton } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+
+const OBSERVABILITY_ONBOARDING_FEEDBACK_LINK =
+  'https://ela.st/logs-onboarding-feedback';
+
+export function ObservabilityOnboardingHeaderActionMenu() {
```

Comment:
(nit) why is it called a menu, when it's only a button or link

Reply:
You are right but it gives space to other actions to come into play there